gui/src/main.cpp

```diff
@@ -1,17 +1,19 @@
 /*
 ** EPITECH PROJECT, 2024
-** Zappy
+** Zappy [WSL: kali-linux]
 ** File description:
-** No file there , just an epitech header example .
-** You can even have multiple lines if you want !
+** GUI-main
 */
 
-#include "GUI.hpp"
+#include <iostream>
+#include "handle_args/HandleArgs.hpp"
+
+using namespace GUI;
 
 int main(int ac, char **av)
 {
-    (void)ac;
-    (void)av;
-    hello_world();
+    GUI::HandleArgs args;
+
+    args.checkArgs(ac, av);
```

Comment:
GUI::HandleArgs#checkArgs gives you a return code, you should exit if it's > 0, no ?

Reply:
It's my bad i already throw exceptions in the checkArgs function, im going to change it to return void